xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+        the <filename>/etc/sudoers.d/</filename> directory. Any file in there is automatically included.
+        To edit a file in that subdirectory, run:
+        </para>
+      <screen>sudo visudo -f /etc/sudoers.d/<replaceable>NAME</replaceable></screen>
+
+      <para>Or with <command>nano</command> as editor:</para>
+      <screen>sudo EDITOR=<replaceable>/usr/bin/nano</replaceable> visudo -f /etc/sudoers.d/<replaceable>NAME</replaceable></screen>
+
+      <note>
+        <title>Ignored files in <filename>/etc/sudoers.d</filename></title>
+        <para>
+          The <literal>#includedir</literal> command in <filename>/etc/sudoers</filename>,
+          used for <filename>/etc/sudoers.d</filename>, ignores files that end in <literal>~</literal>
+          or <emphasis>contain</emphasis> a <literal>.</literal>.
+        </para>
+      </note>
+
+      <para>For more info on the <command>visudo</command> command, run <command>man 8 visudo</command>.</para>
+    </sect2>
+
+    <sect2 xml:id="sec.sudo.conf.syntax">
+      <title>Basic sudoers syntax</title>
+      <para>
+        The most important syntax constructs for <filename>/etc/sudoers</filename>
+        and files under <filename>/etc/sudoers.d</filename> are:
+      </para>
+
+      <screen>
+# Everything on a line after a # gets ignored <co xml:id="co.sudo.syntax.comment"/>
+Defaults !insults # Disable the insults flag <co xml:id="co.sudo.syntax.flag"/>
+Defaults env_keep += "DISPLAY HOME" # Add DISPLAY and HOME to env_keep
+&exampleuser_plain; ALL = NOPASSWD: /usr/bin/frobnicate, PASSWD: /usr/bin/journalctl <co xml:id="co.sudo.syntax.rule"/>
+</screen>
+
+      <calloutlist>
+        <callout arearefs="co.sudo.syntax.comment">
+          <para>
+            There are two exceptions: <literal>#include</literal> and <literal>#includedir</literal> are normal commands. Followed by digits, it specifies a UID.
+          </para>
+        </callout>
+        <callout arearefs="co.sudo.syntax.flag">
+          <para>
+            Remove the <literal>!</literal> to enable the flag.
+          </para>
+        </callout>
+        <callout arearefs="co.sudo.syntax.rule">
+          <para>
+            See <xref linkend="sec.sudo.conf.rule"/>
+          </para>
+        </callout>
+      </calloutlist>
+
+      <table xml:id="tab.adm.sudo.options">
+        <title>Useful flags and options</title>
+        <tgroup cols="3">
+          <thead>
+            <row><entry><para>Option name</para></entry><entry><para>Description</para></entry><entry><para>Example</para></entry></row>
+          </thead>
+          <tbody>
+            <row>
+              <entry>
+                <literal>targetpw</literal>
+              </entry>
+              <entry>
+                <para>
+                  This flag controls whether the invoking user is required to enter the password of the target user (&rootuser;) (ON),
+                  or his own password (OFF). The default is ON.
```

Comment:
~~his own password~~ the user password

_You avoid having to use his/her that way_